src/gql/bookings.js

```diff
@@ -1,8 +1,8 @@
 import { gql } from 'apollo-boost';
 
 export const ROOM_BOOKINGS = gql`
-  query bookings($room: String!) {
-    bookings(data: { room: { number: $room } }) {
+  query bookings($room: String!, $start: String) {
+    bookings(data: { room: { number: $room } }, start: $start) {
```

Comment:
Correct me if I am wrong.

Specifying a `start` for the booking query will try to match the bookings that have the specific `start` time. I do not think this is what we want? We want all the bookings after `start` right?

Reply:
@utkarsh867 no actually, `start` will be passed to phoenix, and phoenix will use `start` as `start_gte` param for prisma. So in general, it returns all the bookings after `start`